scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -1166,17 +1268,58 @@ def delete_part(self, device, part_num):
     ret = execute_command(self, cmd, stdin=stdin_str, verbose=True)
     return ret[1]
 
+def get_fid(self, fids, service, idx):
+    fids_list = []
+    len_fids_list = len(fids)
+
+    # Prepare list of all fids of matching service
+    for i in range(len_fids_list):
+        if fids[i]["name"] == service:
+            fids_list.append(fids[i]["fid"])
+
+    num_fids = len(fids_list)
+    idx = int(idx)
+    if num_fids > 0:
+        if idx < num_fids:
+            return fids_list[idx]
+        else:
+            self.logger.error(f"Invalid index({idx}) of service({service})"
+                              f"Valid index should be in range [0-{num_fids-1}]."
+                              "Returning -1.")
+            return -1
+    else:
+        self.logger.error(f"No fids for service({service}). Returning -1.")
+        return -1
+
+# Fetch fid of service using command 'hctl fetch-fids'
+# First populate a yaml file with the output of command 'hctl fetch-fids'
+# Use this yaml file to get proper fid of required service.
+def fetch_fid(self, service, idx):
+    cmd = "hctl fetch-fids"
+    out = execute_command(self, cmd)
+    self.logger.info(f"Available fids:\n{out[0]}\n")
+    fp = open(TEMP_FID_FILE, "w")
+    fp.write(out[0])
+    fp.close()
+    fp = open(TEMP_FID_FILE, "r")
+    fids = yaml.safe_load(fp)
+    if len(fids) == 0:
+        self.logger.error(f"No fids returned by 'hctl fetch-fids'. Returning -1.\n")
+        return -1
+    fid = get_fid(self, fids, service, idx)
+    return fid
+
 # If service is one of [ios,confd,hax] then we expect fid to start the service
 # and start services using motr-mkfs and motr-server.
 # For other services like 'motr-free-space-mon' we do nothing.
```

Comment:
I think for start free-space-mon .. we need --service name 
and invoke "/usr/libexec/cortx-motr/motr-free-space-monitor" script in that container

Reply:
Done